src/browser/components/CompassItem.styl

```diff
@@ -0,0 +1,9 @@
+.item
+  align-items center
+  display flex
+  flex-direction column
+  width: 33.3%;
+  text-align: center;
```

Comment:
`justify-content`?

Reply:
`flex-grow: 1`?

Reply:
Do you mean flex-grow both with width?  In flexbox probably there's no option to define amount of elements in row.. so, we have to define width of 33.3% for a single one if we want to have 3, am I right?

Reply:
Nope. Only `flex-grow`. [Please see the exellent guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).